fe/fe-core/src/main/java/com/starrocks/sql/optimizer/base/ColumnRefFactory.java

```diff
@@ -73,6 +76,17 @@ public ColumnRefOperator getColumnRef(int id) {
         return columnRefs.get(id - 1);
     }
 
+    public Set<ColumnRefOperator> getColumnRefs(ColumnRefSet columnRefSet) {
+        Preconditions.checkState(!columnRefSet.isEmpty());
```

Comment:
Is this check necessary for sake of generality?

Reply:
You are right. It can remove.